tests/integration/commands/dev/dev.config.test.ts

```diff
@@ -459,7 +459,7 @@ describe.concurrent('commands/dev/config', () => {
     })
   })
 
-  test.runIf(gte(version, '18.14.0'))('should support functions with streaming responses', async (t) => {
+  test.runIf(gte(version, '20.12.2'))('should support functions with streaming responses', async (t) => {
```

Comment:
I don't think we need this anymore because now the lowest version we test with has support for `Request`/`Response`, which is what this condition was about.

```suggestion
  test('should support functions with streaming responses', async (t) => {
```